version.txt

```diff
@@ -0,0 +1 @@
+2.0
```

Comment:
We also have a version defined in auslib/__init__.py, providing auslib.version. Can we squash the two use-cases into one ?